build.sbt

```diff
@@ -230,7 +230,7 @@ lazy val interfaces = project
     moduleName := "mtags-interfaces",
     autoScalaLibrary := false,
     mimaPreviousArtifacts := Set(
-      "org.scalameta" % "mtags-interfaces" % "0.11.10"
+      "org.scalameta" % "mtags-interfaces" % "0.11.11"
```

Comment:
I wish there was just a way to tell mima "last stable", and it would use coursier to just find that version and test against it.

Reply:
Note that in general, comparing against only to the last artifact  (and not all previous artifacts) is not safe and may let breakage slip through.
- https://github.com/lightbend/mima/discussions/724

Reply:
We only use mima for Java interfaces, which we plan to only add new methods to, so should be safe.